src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -40,6 +40,37 @@ pub const INITIAL_DATA_SIZE: DataBlocks = DataBlocks(768);
 const INITIAL_MDV_SIZE: Sectors = Sectors(32 * IEC::Ki); // 16 MiB
 
 
+pub struct ThinPoolSizeParams {
+    meta_size: MetaBlocks,
+    data_size: DataBlocks,
+    mdv_size: Sectors,
+}
+
+impl ThinPoolSizeParams {
+    /// The number of Sectors in the MetaBlocks.
+    pub fn meta_size(&self) -> Sectors {
+        self.meta_size.sectors()
+    }
+    /// The number of Sectors in the DataBlocks.
+    pub fn data_size(&self) -> Sectors {
+        self.data_size.0 * DATA_BLOCK_SIZE
```

Comment:
Please dereference here. Ideally, the field would be private outside the module in which its defined. But due to some compiler constraints, which we put up with so we can define Sectors constants, it is not. Eventuallly, we hope that these constraints will go away, and when they do, the compiler will complain about accessing the .0 field out side the module. Meanwhile, we just have to pretend.

Reply:
Your comment is about self.data_size.0?  I thought the rule was .0 if inside implementation * if outside?

Reply:
I'm not sure what you're saying there. But, pretend it's a regular struct w/ a private field. You could access that private field within the module in which that struct was defined, which for Sectors, is the types module, but you could not access that field outside the types module. If you follow this rule, then you should not access the field here.

Reply:
ok